workflow-dev/tpl/deis-registry-token-refresher.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: deis-registry-token-refresher
+  namespace: deis
+  labels:
+    heritage: deis
+spec:
+  replicas: 1
+  selector:
+    heritage: deis
+    app: deis-registry-token-refresher
+  template:
+    metadata:
+      labels:
+        heritage: deis
+        app: deis-registry-token-refresher
+    spec:
+      containers:
+        - name: deis-registry-token-refresher
+          image: kmala/registry-token-refresher:canary
```

Comment:
currently no...i will change it once this PRhttps://github.com/deis/registry-token-refresher/pull/1 is merged